learn/work/fairness-readmission/index.qmd

```diff
@@ -0,0 +1,516 @@
+
+This article will demonstrate a fairness-oriented workflow for training a machine learning model to identify high-risk patients. Throughout the model development process, we'll consider the social impacts that different modeling decisions may have when such a model is deployed in context.
+
+## Setup
+
+The data we'll use in this analysis is a publicly available database containing information on `r format(nrow(readmission), big.mark = ",")` hospital stays from diabetes patients. The data comes from a study by @strack2014, where the authors model the effectiveness of a particular lab test in predicting readmission. A version of that data is available in the [readmission R package](https://simonpcouch.github.io/readmission/):
+
+```{r}
+#| label: "load-readmission"
+#| eval: true
+library(readmission)
+
+readmission
+```
+
+The first variable in this data, `readmitted`, gives whether the patient was readmitted within 30 days of discharge. We'll use this variable as a proxy for "unmet need for additional care," in that readmission within one month indicates that the patient may have benefited from additional attention during their hospital stay; if a machine learning model consistently identifies lesser need (via prediction of non-readmission) in one subgroup than another, the subgroup allocated lesser need will go without care they'd benefit from. We'd like to train a model that is both fair with regard to how it treats `race` groups and is as performant as possible. The tidymodels framework provides the tools needed to identify such disparities.
+
+Loading needed packages:
+
+```{r}
+#| label: "load-tidymodels"
+library(tidymodels)
+library(baguette)
+library(desirability2)
+library(GGally)
+```
+
+## Exploratory Analysis
+
+Let's start off our analysis with some explanatory summarization and plotting. First, taking a look at our outcome variable:
+
+```{r}
+#| label: "count-by-readmitted"
+readmission %>%
+  count(readmitted)
+```
+
+`r round(mean(readmission$readmitted == "Yes"), 3)*100`% of patients were readmitted within 30 days after being discharged from the hospital. This is an example of a modeling problem with a _class imbalance_, where one value of the outcome variable is much more common than another. Now, taking a look at the counts of those in each protected class:
+
+```{r}
+#| label: "count-by-race"
+readmission %>%
+  count(race)
+```
+
+::: callout-note
+We'll refer to the `race` groups in this data by their actual value (e.g. `"Caucasian"` rather than Caucasian) so as to not to take for granted the choices that the dataset authors made in choosing these categorizations. Racial categorizations are not stable across time and place---as you read on, consider how a change in the categories used in data collection might affect this analysis [@omi1994].
```

Comment:
Ah, because there's a typo!😄 Thank you. [e21af27](https://github.com/tidymodels/tidymodels.org/pull/52/commits/e21af27668c751d5159b7f0539c321000860814f).